### ISSUE TITLE
purefb_connect module not allow to disable encryption

### ISSUE THREAD

**Issue by mahendra-maplelabs** [Pure-Storage-Ansible/FlashBlade-Collection#94]
**Describe the bug**
purefb_connect module not allowing to disable encryption.

**To Reproduce**
1. Create array connection with `encrypted: true` parameter
2. Try to delete by using `encrypted: false` parameter
3. See error

**Sample playbook**
```
---
- name: FB Array connection
  hosts: localhost
  gather_facts: false
  collections:
    - purestorage.flashblade
  tasks:
  - name: Create array connection
    purefb_connect:
      state: present
      encrypted: false
      target_url: 10.21.xxx.11
      target_api: T-6773eccd-xxx-xxxx-xxxx-8db062fb8b6a
      fb_url: 10.21.xxx.151
      api_token: T-0b8ad89c-xxxx-xxxx-xxxx-286607dc2cd2
```

**Error:**

```
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to change encryption setting for array connection."}
```

**Expected behavior**
Switching parameter from `encrypted: true` to `encrypted: false` should disable encryption.

Platform versions:
purity-fb version v1.9.2
collection - latest code
Ansible version 2.10

**Comment by sdodsley** [comment]
I need to get 2 uncnnected FBs to resolve this - give me a few days to get these ready

**Comment by sdodsley** [comment]
Addressed by #96. 
Please check if this resolves the problem.

**Comment by mahendra-maplelabs** [comment]
This resolves my issue. Thanks @sdodsley
